src/common/options/osd.yaml.in

```diff
@@ -214,7 +214,7 @@ options:
 - name: osd_repair_during_recovery
   type: bool
   level: advanced
-  desc: Allow requested repairing when PGs on the OSD are undergoing recovery
+  desc: not in use
```

Comment:
Can we just remove it?  Do we need to leave it as "not in use"?

Reply:
Yes - I'll remove it.